python/lsst/ip/isr/crosstalk.py

```diff
@@ -23,17 +23,441 @@
+
+        return outDict
+
+    @classmethod
+    def fromTable(cls, tableList):
+        """Construct calibration from a list of tables.
+
+        This method uses the `fromDict` method to create the
+        calibration, after constructing an appropriate dictionary from
+        the input tables.
+
+        Parameters
+        ----------
+        tableList : `list` [`lsst.afw.table.Table`]
+            List of tables to use to construct the crosstalk
+            calibration.
+
+        Returns
+        -------
+        calib : `lsst.ip.isr.CrosstalkCalib`
+            The calibration defined in the tables.
+
+        """
+        coeffTable = tableList[0]
+
+        metadata = coeffTable.meta
+        inDict = dict()
+        inDict['metadata'] = metadata
+        inDict['hasCrosstalk'] = metadata['HAS_CROSSTALK']
+        inDict['nAmp'] = metadata['NAMP']
+
+        inDict['coeffs'] = coeffTable['CT_COEFFS']
+        if len(tableList) > 1:
+            inDict['interChip'] = dict()
+            interChipTable = tableList[1]
+            for record in interChipTable:
+                inDict['interChip'][record['IC_SOURCE_DET']] = record['IC_COEFFS']
+
+        return cls().fromDict(inDict)
+
+    def toTable(self):
+        """Construct a list of tables containing the information in this calibration.
+
+        The list of tables should create an identical calibration
+        after being passed to this class's fromTable method.
+
+        Returns
+        -------
+        tableList : `list` [`lsst.afw.table.Table`]
+            List of tables containing the crosstalk calibration
+            information.
+
+        """
+        tableList = []
+        self.updateMetadata()
+        catalog = Table([{'CT_COEFFS': self.coeffs.reshape(self.nAmp*self.nAmp)}])
+        catalog.meta = self.getMetadata().toDict()
+        tableList.append(catalog)
+
+        if self.interChip:
+            interChipTable = Table([{'IC_SOURCE_DET': sourceDet,
+                                     'IC_COEFFS': self.interChip[sourceDet].reshape(self.nAmp*self.nAmp)}
+                                    for sourceDet in self.interChip.keys()])
+            tableList.append(interChipTable)
+        return tableList
+
+    # Implementation methods.
+    def extractAmp(self, image, amp, ampTarget, isTrimmed=False):
+
+        """Extract the image data from an amp, flipped to match ampTarget.
+
+        Parameters
+        ----------
+        image : `lsst.afw.image.Image` or `lsst.afw.image.MaskedImage`
+            Image containing the amplifier of interest.
+        amp : `lsst.afw.cameraGeom.Amplifier`
+            Amplifier on image to extract.
+        ampTarget : `lsst.afw.cameraGeom.Amplifier`
+            Target amplifier that the extracted image will be flipped
+            to match.
+        isTrimmed : `bool`
+            The image is already trimmed.
+            This should no longer be needed once DM-15409 is resolved.
```

Comment:
Please add an explicit TODO here (per https://developer.lsst.io/python/style.html?highlight=todo#to-do-comments-should-include-a-jira-issue-key)